noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -269,6 +269,7 @@ impl PrivateContext {
                 // for the correct public key has been received.
                 get_key_validation_request(pk_m_hash, key_index)
             };
+            assert_eq(request.pk_m.hash(), pk_m_hash, "Obtained invalid key validation request");
```

Comment:
I find `assert_eq` nicer to read than `assert(left == right)` so I sneaked this change in here and in other places.

I think we didn't use it before as it was not yet implemented.

Reply:
This assert is not present in the origianl code though?

Reply:
It actually [is present](https://github.com/AztecProtocol/aztec-packages/blob/c2c0bc697312fe4f63337d66b7419b662d9f55ee/noir-projects/aztec-nr/aztec/src/context/private_context.nr#L268). I have no idea why github fails to show the diff here.

Reply:
Turns out I am an idiot and I've accidentally deleted the assert in a PR down the stack while checking for its presence on master 🤦

Reverted that change and the diff is looking fine now